### DISCUSSION TITLE
YT Channel Downloader v0.4.0

### DISCUSSION THREAD

**Discussion by hyperfield** [hyperfield/yt-channel-downloader#10] [Announcements]
The latest downloadable installer of the release of YT Channel Downloader for Windows (compiled from the updated repository). This installer also includes the uninstaller and a script to download and install ffmpeg on your system if it's not present. ffmpeg is a set of libraries and tools for multimedia processing, and it is needed for YT Channel Downloader to work properly.

[Download the latest Windows installer here](https://quicknode.net/media/dist/YT_Channel_Downloader_0_4_0_installer.zip)

<hr /><em>This discussion was created from the release <a href='https://github.com/hyperfield/yt-channel-downloader/releases/tag/'>YT Channel Downloader v0.4.0</a>.</em>

**Comment by polin79** [comment]
Hi,
I don't understand why, but I'm having problems in YouTube downloads. Till last week non problem and now I'm having problems: downloads don't start

Paolo
